app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/table.tsx

```diff
@@ -103,7 +103,7 @@ export const renderTable = ({
           {numNotSupported} not or partially supported {'  '}
         </WarningMessage>
         <SmallButton variant="text" color="primary" onClick={() => handleShowErrClick()}>
-          {filterErrs.length ? 'HIDE' : 'SHOW'}
+          {filterErrs.length ? 'hide' : 'show'}
```

Comment:
Sorry, should have said: Initial caps are fine. But leave all-caps to CSS, since that's something that can be controlled by the design system.

Reply:
yeah i think material ui does this by default on all of its buttons